src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -464,7 +464,7 @@ public void install_noop(ChocolateyConfiguration config, Action<PackageResult> c
 
                 if (installedPackage != null && version != null && version < installedPackage.Version && !config.AllowMultipleVersions && !config.AllowDowngrade)
                 {
-                    string logMessage = "A newer version of {0} (v{1}) is already installed.{2} Use --allow-downgrade or --force to attempt to install older versions, or use side by side to allow multiple versions.".format_with(installedPackage.Id, installedPackage.Version, Environment.NewLine);
+                    string logMessage = "A newer version of {0} (v{1}) is already installed.{2} Use --allow-downgrade or --force to attempt to install older versions, or use --sidebyside to allow multiple versions.".format_with(installedPackage.Id, installedPackage.Version, Environment.NewLine);

```

Comment:
While this looks great, can we get it as `--side-by-side` or `--allow-multiple-versions`?

Reply:
I'm not following. 
As in:
`install older versions, or use --side-by-side to --allow-multiple-versions.".`

Because I'm not sure if that last one is a switch

Reply:
@Mauville he is talking about changing `--sidebyside` to either `--side-by-side` or to `--allow-multiple-versions`.

For instance:
`or use --side-by-side to allow multiple versions`
or
`or use --allow-multiple-versions to allow multiple versions`

Reply:
Ok. I feel like --side-by-side would be a better alternative in this case because syntactically speaking, it makes more sense to pass a flag like

choco install anki --side-by-side

than 

choco install anki --allow-multiple-versions